### PR TITLE
Seed phrase screen new design

### PR DESCRIPTION
**Pull request by OldCrab**

### CODE REVIEW

File: p2p_wallet/Scenes/CreateOrRestoreWallet/RestoreWallet/EnterSeedViewController/EnterSeedInfo/EnterSeedInfoNavigationBar.swift [hunk 1]
```diff
@@ -0,0 +1,38 @@
+//
+//  EnterSeedInfoNavigationBar.swift
+//  p2p_wallet
+//
+//  Created by Andrew Vasiliev on 18.11.2021.
+//
+
+import UIKit
+
+final class EnterSeedInfoNavigationBar: WLNavigationBar {
```
Comment (bigearsenal) [inline p2p_wallet/Scenes/CreateOrRestoreWallet/RestoreWallet/EnterSeedViewController/EnterSeedInfo/EnterSeedInfoNavigationBar.swift:10]: subview of a scene must be implemented in scene's extension
```swift
extension EnterSeedInfo {
    final class NavigationBar: WLNavigationBar
}
```
The same for EnterSeedNavigationBar

File: p2p_wallet/Scenes/CreateOrRestoreWallet/RestoreWallet/EnterSeedViewController/EnterSeedMainButtonContent.swift [hunk 2]
```diff
@@ -0,0 +1,16 @@
+//
+//  EnterSeedMainButtonContent.swift
+//  p2p_wallet
+//
+//  Created by Andrew Vasiliev on 18.11.2021.
+//
+
+enum EnterSeedMainButtonContent {
```
Comment (bigearsenal) [inline p2p_wallet/Scenes/CreateOrRestoreWallet/RestoreWallet/EnterSeedViewController/EnterSeedMainButtonContent.swift:8]: Enum that releated to a scene should be defined inside that scene.
Let's add this enum MainButtonContent inside EnterSeed.swift, after NavigatableScene, the same for InvalidationReason.
Avoid polluting global namespace

**Review by bigearsenal — CHANGES_REQUESTED** [review]

**Comment by bigearsenal** [comment]
ADDITIONAL TASK: https://p2pvalidator.atlassian.net/browse/P2PW-1088
image

File: p2p_wallet/Scenes/CreateOrRestoreWallet/RestoreWallet/EnterSeedViewController/EnterSeed.RootView.swift [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+        let disposeBag = DisposeBag()
+        
+        // MARK: - Properties
+        private let viewModel: EnterSeedViewModelType
+
+        // MARK: - Subviews
+        lazy var scrollView = ContentHuggingScrollView(scrollableAxis: .vertical, contentInset: .init(only: .bottom, inset: 40))
+        lazy var stackView = UIStackView(axis: .vertical, alignment: .fill, distribution: .fill)
+        private let textView = ExpandableTextView()
+        private let agreeTermsAndConditionsView = AgreeTermsAndConditionsView()
```
Comment (bigearsenal) [inline p2p_wallet/Scenes/CreateOrRestoreWallet/RestoreWallet/EnterSeedViewController/EnterSeed.RootView.swift:23]: AgreeTermsAndConditionsView.swift file not found

**Review by bigearsenal — APPROVED** [review]

File: p2p_wallet/Scenes/CreateOrRestoreWallet/CreateWallet/CreateSecurityKeys/CreateSecurityKeys.RootView.swift [hunk 4]
```diff
@@ -31,11 +31,16 @@ extension CreateSecurityKeys {
         private let verifyManualButton: WLStepButton = WLStepButton.sub(text: L10n.verifyManually)
         
         private let keysView: KeysView = KeysView()
-        private let keysViewAction: KeysViewActions = KeysViewActions()
+        private let keysViewActions: KeysViewActions = KeysViewActions()
+        private let agreeTermsAndConditions = AgreeTermsAndConditionsView()
```
Comment (bigearsenal) [inline p2p_wallet/Scenes/CreateOrRestoreWallet/CreateWallet/CreateSecurityKeys/CreateSecurityKeys.RootView.swift:35]: image
AgreeTermsAndConditionsView: file not found

**Review by bigearsenal — CHANGES_REQUESTED** [review]

**Review by bigearsenal — CHANGES_REQUESTED** [review]
remove THPinViewController from your local repository
```rm -rf THPinViewController```

**Review by bigearsenal — CHANGES_REQUESTED** [review]
The TermsAndConditionsVC has been removed from develop, so let's remove `TermsAndConditionsVC`, use `WLMarkdownVC(title: L10n.termsOfUse.uppercaseFirst, bundledMarkdownTxtFileName: "Terms_of_service")` instead

**Comment by OldCrab** [comment]
> 



> The TermsAndConditionsVC has been removed from develop, so let's remove `TermsAndConditionsVC`, use `WLMarkdownVC(title: L10n.termsOfUse.uppercaseFirst, bundledMarkdownTxtFileName: "Terms_of_service")` instead

Can't find a place to remove, I thought I've already done it

**Comment by bigearsenal** [comment]
> > 
> 
> > The TermsAndConditionsVC has been removed from develop, so let's remove `TermsAndConditionsVC`, use `WLMarkdownVC(title: L10n.termsOfUse.uppercaseFirst, bundledMarkdownTxtFileName: "Terms_of_service")` instead
> 
> Can't find a place to remove, I thought I've already done it

Oh. I see, the file has been removed from the .xcodeproj, but still remains on repository.
I will approve this PR and remove upon merging. Thanks

**Review by bigearsenal — APPROVED** [review]

**Pull request merged**